### PR TITLE
Editing hub documents

### PR DESCRIPTION
**Pull request by EMcWhinn**
Editing hub documents to align them with style guide

Minor edits to hub documentation

https://issues.redhat.com/browse/AAP-10816

### CODE REVIEW

File: downstream/modules/automation-hub/proc-edit-namespace.adoc [hunk 1]
```diff
@@ -4,14 +4,14 @@
 
 = Adding additional information and resources to a namespace
 
-You can add information and provide resources for your users to accompany collections included in the namespace. Add a logo, description and link users to your GitHub repository, issue tracker or other online assets. You can also enter markdown text in the *Edit resources* tab to include additional information helpful to end users who may use your collection in their automation tasks.
+You can add information and provide resources for your users to accompany collections included in the namespace. Add a logo, a description and link users to your GitHub repository, issue tracker, or other online assets. You can also enter markdown text in the *Edit resources* tab to include more information. This is helpful to end users who may use your collection in their automation tasks.
```
Comment (emurtoug) [inline downstream/modules/automation-hub/proc-edit-namespace.adoc:7]: The syntax here is a little confusing. Would 
'Add a logo and a description, and link users to your GitHub repository, issue tracker, or other online assets.' work?

**Review by emurtoug — COMMENTED** [review]
Hi Ed! One comment for you there, otherwise LGTM. Thanks!

File: downstream/modules/automation-hub/proc-edit-namespace.adoc (hunk 1 above)
Comment (EMcWhinn) [reply downstream/modules/automation-hub/proc-edit-namespace.adoc]: Hey Lizzi, thanks for reviewing. I agree with the syntax suggested and updated. Thanks!

File: downstream/assemblies/hub/assembly-working-with-namespaces.adoc [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
-= Using namespaces to manage collections in Automation Hub
+= Using namespaces to manage collections in {HubName}
 
-You can use namespaces in Automation Hub to organize collections developed within your organization for internal distribution and use.
+You can use namespaces in {HubName} to organize collections developed within your organization for internal distribution and use.
 
-Working with namespaces requires a group that has permissions to create, edit and upload collections to namespaces. Collections uploaded to a namespace may require administrative approval before you can publish them and make them available for use. See https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/1.0/html/managing_user_access_in_automation_hub/index[Managing user access in Automation Hub] for more information on groups and permissions to determine how to configure Automation Hub for your content curators and developers.
+Working with namespaces requires a group that has permissions to create, edit and upload collections to namespaces. Collections uploaded to a namespace may require administrative approval before you can publish them and make them available for use.
```
Comment (ianf77) [inline downstream/assemblies/hub/assembly-working-with-namespaces.adoc:7]: Collections uploaded to a namespace can require ...

File: downstream/assemblies/hub/assembly-working-with-namespaces.adoc [hunk 3]
```diff
@@ -14,7 +14,7 @@ include::automation-hub/proc-create-namespace.adoc[leveloffset=+1]
 
 include::automation-hub/proc-edit-namespace.adoc[leveloffset=+1]
 
-Once a namespace is created, groups with permissions to upload to it can start adding their collections for approval. Upon approval, collections in the namespace will appear in the *Published* repository.
+When you create a namespace, groups with permissions to upload to it can start adding their collections for approval. Upon approval, collections in the namespace will appear in the *Published* repository.
```
Comment (ianf77) [inline downstream/assemblies/hub/assembly-working-with-namespaces.adoc:17]: Collections in the namespace appear in the Published repository after approval.

File: downstream/modules/automation-hub/proc-create-content-developers.adoc [hunk 4]
```diff
@@ -4,33 +4,33 @@
...
-You can create a new group in Automation Hub designed to support content curation in your organization who will contribute internally developed collections for publication in Automation Hub.
+You can create a new group in {HubName} designed to support content curation in your organization that contributes internally developed collections for publication in {HubName}.
 
-In this section you will create a new group and assign the required permissions to help content developers create namespaces and upload their collections to Automation Hub.
+This section shows you how to create a new group and assign the required permissions to help content developers create namespaces and upload their collections to {HubName}.
 
 .Prerequisites
 
-* You have *admin* permissions in Automation Hub and create groups.
+* You have *admin* permissions in {HubName} and create groups.
```
Comment (ianf77) [inline downstream/modules/automation-hub/proc-create-content-developers.adoc:13]: Is it administrative, or administration? Or is admin the name given on the interface?

File: downstream/modules/automation-hub/proc-delete-namespace.adoc [hunk 5]
```diff
@@ -4,18 +4,18 @@
...
 . Navigate to menu:Collections[Namespaces].
 . Click the namespace to be deleted.
 . Click image:more_actions.png[More actions], then click btn:[Delete namespace].
 +
-NOTE: If the btn:[Delete namespace] button is disabled, it means that this namespace contains a collection with dependencies. Review the collections in this namespace and delete any dependencies to proceed with the namespace deletion. See _Managing collections on automation hub_ in the link:https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform[Red Hat Ansible Automation Platform documentation] for information about deleting collections.
+NOTE: If the btn:[Delete namespace] button is disabled, it means that this namespace contains a collection with dependencies. Review the collections in this namespace and delete any dependencies to proceed with the namespace deletion. See link:https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/2.3/html-single/managing_red_hat_certified_and_ansible_galaxy_collections_in_automation_hub/index[Managing Red Hat Certified and Ansible Galaxy collections in automation hub] for information about deleting collections.
```
Comment (ianf77) [inline downstream/modules/automation-hub/proc-delete-namespace.adoc:19]: Think this is the wrong document, there's nothing about deleting collections in there. You probably want Uploading content to Red Hat Automation Hub, Chapter 2 is called 'Deleting a collection on automation hub'

File: downstream/modules/automation-hub/proc-delete-namespace.adoc [hunk 6]
```diff
@@ -4,18 +4,18 @@
...
 
-The deleted namespace, as well as its associated collections, will now be deleted and removed from the namespace list view.
+The namespace that you deleted, as well as its associated collections, is now deleted and removed from the namespace list view.
```
Comment (ianf77) [inline downstream/modules/automation-hub/proc-delete-namespace.adoc:21]: Although if it's a namespace you're deleting, you want Section 1.7 of Curating collections using namespaces in Automation Hub.

File: downstream/modules/automation-hub/proc-edit-namespace.adoc [hunk 7]
```diff
@@ -4,14 +4,14 @@
 
 = Adding additional information and resources to a namespace
 
-You can add information and provide resources for your users to accompany collections included in the namespace. Add a logo, description and link users to your GitHub repository, issue tracker or other online assets. You can also enter markdown text in the *Edit resources* tab to include additional information helpful to end users who may use your collection in their automation tasks.
+You can add information and provide resources for your users to accompany collections included in the namespace. Add a logo and a description, and link users to your GitHub repository, issue tracker, or other online assets. You can also enter markdown text in the *Edit resources* tab to include more information. This is helpful to end users who may use your collection in their automation tasks.
```
Comment (ianf77) [inline downstream/modules/automation-hub/proc-edit-namespace.adoc:7]: This is helpful to users who use your collection...

File: downstream/modules/automation-hub/proc-uploading-collections.adoc [hunk 8]
```diff
@@ -4,7 +4,7 @@
 
 = Uploading collections to your namespaces
 
-You can upload internally developed collections to your local Automation Hub namespace for review and approval by an Automation Hub administrator. Once approved, the collection will move to the *Published* content repository where Automation Hub users can view and download it.
+You can upload internally developed collections to your local {HubName} namespace for review and approval by an {HubName} administrator. Once approved, the collection will move to the *Published* content repository where {HubName} users can view and download it.
```
Comment (ianf77) [inline downstream/modules/automation-hub/proc-uploading-collections.adoc:7]: When approved, the collection

File: downstream/modules/automation-hub/proc-create-content-developers.adoc (hunk 4 above)
Comment (EMcWhinn) [reply downstream/modules/automation-hub/proc-create-content-developers.adoc:13]: > Is it administrative, or administration? Or is admin the name given on the interface?

Is there a way that we can confirm this UI?

**Pull request merged**